ludwig/schema/decoders/utils.py

```diff
@@ -60,7 +60,6 @@ def _jsonschema_type_mapping():
                 },
                 "title": "decoder_options",
                 "allOf": get_decoder_conds(feature_type),
-                "required": ["type"],
```

Comment:
Is this something we'll need to bring back once we're using Ludwig 0.6 throughout our entire ecosystem? With the removal of `type` being required, this will affect both the JSON Schema we generate and the docs right?

Reply:
So I thought about it a bit and we actually don't need to require type from the user on their config. Reason being, the default will just get filled in for them if they don't specify. So if they were to say just specify dropout, it would just switch out the dropout value for the default encoder.